antarest/core/configdata/repository.py

```diff
@@ -10,13 +10,13 @@
 #
 # This file is part of the Antares project.
 
-import json
 from operator import and_
 from typing import Optional
 
 from antarest.core.configdata.model import ConfigData
 from antarest.core.jwt import DEFAULT_ADMIN_USER
 from antarest.core.model import JSON
+from antarest.core.serialization.utils import from_json, to_json
```

Comment:
I was more thinking of removing the `utils` part (just antarest.core.serialization)